apps/gallery/index.html

```diff
@@ -57,7 +57,6 @@
     <link rel="stylesheet" type="text/css" href="app://theme.gaiamobile.org/shared/elements/gaia-theme/gaia-theme.css">
     <link rel="stylesheet" type="text/css" href="shared/elements/gaia-icons/gaia-icons.css" />
     <link rel="stylesheet" type="text/css" href="shared/elements/gaia-icons/bidi-helper.css" />
-    <script defer src="shared/elements/gaia-header/dist/gaia-header.js"></script>
```

Comment:
https://github.com/mozilla-b2g/gaia/blob/master/apps/gallery/open.html#L38
phew, this is why the patch is working even when we don't have gaia-header shared asset not commented in index.html.